uk.co.compucorp.civicrm.hrcore/tests/phpunit/CRM/HRCore/APIWrapper/DefaultLimitRemoverTest.php

```diff
@@ -21,43 +29,37 @@ public function setUpHeadless() {
   }
 
   public function setUp() {
-    $this->defaultLimitRemoverObj = new CRM_HRCore_APIWrapper_DefaultLimitRemover();
-
-    $this->testParameters['params'] = [
-      'action' => 'get',
-      'entity' => 'contact',
-      'id' => 1,
-    ];
+    $this->defaultLimitRemover = new DefaultLimitRemover();
   }
 
-  public function testFromApiInputWithLimitSetInParameters() {
+  public function testTheDefaultLimitShouldNotBeAppliedIfASpecificLimitHasBeenSet() {
     $this->testParameters['params']['options']['limit'] = 10;
 
-    $actualValue =  $this->defaultLimitRemoverObj->fromApiInput($this->testParameters);
+    $actualValue =  $this->defaultLimitRemover->fromApiInput($this->testParameters);
 
     $this->assertEquals(
       $this->testParameters['params']['options']['limit'],
       $actualValue['params']['options']['limit']
     );
   }
 
-  public function testFromApiInputWithLimitNotSetInParameters() {
-    $actualValue =  $this->defaultLimitRemoverObj->fromApiInput($this->testParameters);
+  public function testTheDefaultNoLimitValueShouldBeAppliedIfNoLimitHasBeenSet() {
```

Comment:
I meant here "Default No Limit Value" which is 0  " .. but seems like its not very clear like that so I will change it